Use <code> instead of <tt> for Javadoc

In files generated by Jflex 1.7.0 there pieces of Javadoc that look like this:
```java
  /**                                                                           
   * Returns the character at position <tt>pos</tt> from the                    
   * matched text.                                                              
   *                                                                            
   * It is equivalent to yytext().charAt(pos), but faster                       
   *                                                                            
   * @param pos the position of the character to fetch.                         
   *            A value from 0 to yylength()-1.                                 
   *                                                                            
   * @return the character at position pos                                      
   */                                                                           
  public final char yycharat(int pos) {                                         
    return zzBuffer[zzStartRead+pos];                                           
  }                            
```
This fails javadoc checks in JDK 11 (and thus blocking oracle/opengrok#2522), e.g.:
```
[ERROR] /home/travis/build/vladak/OpenGrok/opengrok-indexer/target/generated-sources/jflex/org/opengrok/indexer/search/context/PlainLineTokenizer.java:46: error: tag not supported in the generated HTML version: tt
[ERROR]  * from the specification file <tt>/home/travis/build/vladak/OpenGrok/opengrok-indexer/target/jflex-sources/PlainLineTokenizer.lex</tt>
```
It seems that using `<code>` instead of `<tt>` should fix the problem.

--- a/jflex/examples/cup-interpreter/src/main/java/Texp.java
+++ b/jflex/examples/cup-interpreter/src/main/java/Texp.java
@@ -13,9 +13,9 @@
  * <p>The non terminal exp is the sum of multiple variants and therefore modelled as an abstract
  * class.
  *
- * <p>The interpretation function <tt>interpret</tt> is called with valuations of input variables
- * <tt>in</tt> and parameters <tt>par</tt>. Before interpret can be called, pointers and variable
- * indices must be set with <tt>prepInterp</tt>.
+ * <p>The interpretation function {@code interpret} is called with valuations of input variables
+ * {@code in} and parameters {@code par}. Before interpret can be called, pointers and variable
+ * indices must be set with {@code prepInterp}.
  */
 abstract class Texp implements AST {
   // test context conditions (DefFun,DefVar,Arity)

--- a/jflex/src/main/java/jflex/generator/Emitter.java
+++ b/jflex/src/main/java/jflex/generator/Emitter.java
@@ -122,7 +122,7 @@ public final class Emitter {
    * if the file already exists.
    *
    * @param name the name (without path) of the file
-   * @param input fall back location if path = <tt>null</tt> (expected to be a file in the directory
+   * @param input fall back location if {@code path = null} (expected to be a file in the directory
    *     to write to)
    * @return The constructed File
    */

--- a/jflex/src/main/jflex/skeleton.nested
+++ b/jflex/src/main/jflex/skeleton.nested
@@ -280,7 +280,7 @@
    *
    * All internal variables are reset, the old input stream
    * <b>cannot</b> be reused (internal buffer is discarded and lost).
-   * Lexical state is set to <tt>ZZ_INITIAL</tt>.
+   * Lexical state is set to {@code ZZ_INITIAL}.
    *
    * Internal scan buffer is resized down to its initial length, if it has grown.
    *
@@ -356,7 +356,7 @@
 
 
   /**
-   * Returns the character at position <tt>pos</tt> from the
+   * Returns the character at position {@code pos} from the
    * matched text.
    *
    * It is equivalent to yytext().charAt(pos), but faster

--- a/jflex/src/main/jflex/skeleton.nested-1.8.0
+++ b/jflex/src/main/jflex/skeleton.nested-1.8.0
@@ -280,7 +280,7 @@
    *
    * All internal variables are reset, the old input stream
    * <b>cannot</b> be reused (internal buffer is discarded and lost).
-   * Lexical state is set to <tt>ZZ_INITIAL</tt>.
+   * Lexical state is set to {@code ZZ_INITIAL}.
    *
    * Internal scan buffer is resized down to its initial length, if it has grown.
    *
@@ -356,7 +356,7 @@
 
 
   /**
-   * Returns the character at position <tt>pos</tt> from the
+   * Returns the character at position {@code pos} from the
    * matched text.
    *
    * It is equivalent to yytext().charAt(pos), but faster

